### ISSUE TITLE
Not working anymore ?

### ISSUE THREAD

**Issue by gvkyx** [dgoguerra/pokego-scan#9]
since a couple of hours its not working anymore. 

it never goes into the function

```
pokegoScan(coords, function(err, pokemons) {
})
```

**Comment by donedgardo** [comment]
PokeVision was under heavy mantaince today, they twitter feed is blaming us for the scrapping.

<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">We&#39;re working on some fixes. Unfortunately, there are other people scraping our site which severely deteriorates our service.</p>&mdash; Pokevision (@PokeVisionGo) <a href="https://twitter.com/PokeVisionGo/status/757679751561478145">July 25, 2016</a></blockquote>

<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>

I find it funny since they are also scrapping NianticLabs API.

**Comment by gevky** [comment]
Hahaha no way to do it directly from Niantic labs? The code for doing it directly is not open? 

Does it work for you now? It's still dead probably they changed something on their end

**Comment by dgoguerra** [comment]
Its actually working for me. Are you receiving an error? Which one?

**Comment by donedgardo** [comment]
Its working for me 2 now.
Edit: Pokemon Go Under Maintance again possible server load.

**Comment by gevky** [comment]
I didnt throw an error but i just added and true its throwing that error... 

*edit: never mind heroku issue.

**Comment by dannybess** [comment]
Anyway to bypass Cloudflare client side? (calling /map/scan and then /map/data through an HTTP networking library).

**Comment by donedgardo** [comment]
Api still working after cloudflare.

**Comment by gevky** [comment]
Yeah it still works just need to make sure to create some mechanism that it will trigger it again.
Btw do you check for duplicate objects in the pokemons array ? (the array that is returned)
